core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2804,6 +2804,17 @@ payment.f2f.info.openURL=Open web page
 payment.f2f.offerbook.tooltip.countryAndCity=Country and city: {0} / {1}
 payment.f2f.offerbook.tooltip.extra=Additional information: {0}
 
+# Japan Bank Transfer
+payment.japan.bank=Bank or Financial Institution
+payment.japan.bank.select=Search for Financial Institution
+payment.japan.bank.code=Bank Code
+payment.japan.bank.name=Bank Name
+payment.japan.bank.branch.code=Branch Code
```

Comment:
We do have a `payment.branchNr` -> Branch no., but I'm not sure if it is the perfect replacement in that case.